mail/views_test.py

```diff
@@ -0,0 +1,79 @@
+"""
+Tests for HTTP email API views
+"""
+from unittest.mock import Mock, patch
+from django.core.urlresolvers import reverse
+from django.db.models.signals import post_save
+from rest_framework.test import APITestCase
+from rest_framework.response import Response
+from factory.django import mute_signals
+
+from profiles.factories import ProfileFactory
+from courses.factories import ProgramFactory
+from roles.models import Role
+from roles.roles import Staff
+
+
+@patch('mail.views.prepare_and_execute_search')  # pylint: disable=missing-docstring
+@patch('mail.views.MailgunClient')  # pylint: disable=missing-docstring
+class MailViewsTests(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.mail_url = reverse('mail_api')
+        cls.program = ProgramFactory.create(live=True)
+        # create a user with a role for one program
+        with mute_signals(post_save):
+            staff_profile = ProfileFactory.create()
+            cls.staff = staff_profile.user
+        Role.objects.create(
+            user=cls.staff,
+            program=cls.program,
+            role=Staff.ROLE_ID
+        )
+        cls.request_data = {
+            'search_request': {},
+            'email_subject': 'email subject',
+            'email_body': 'email body'
+        }
+
+    def setUp(self):
+        super(MailViewsTests, self).setUp()
+        self.client.force_login(self.staff)
+
+    def test_send_view(self, mock_mailgun_client, mock_prepare_exec_search):
+        """
+        Test that the email send view will accept and return expected values
+        """
+        email_results = ['a@example.com', 'b@example.com']
+        mock_prepare_exec_search.return_value = email_results
+        mock_mailgun_client.send_bcc.return_value = Mock(spec=Response, status_code=200)
```

Comment:
Can you use the enums in `rest_framework.status` to replace the HTTP status codes in this file?